spec/features/idv/steps/in_person/verify_info_spec.rb

```diff
@@ -135,4 +135,67 @@
     # phone page
     expect(page).to have_content(t('titles.idv.phone'))
   end
+
+  context 'with in person verify info controller enabled ' do
+    let(:capture_secondary_id_enabled) { true }
+    let(:enrollment) { InPersonEnrollment.new(capture_secondary_id_enabled:) }
+    let(:user) { user_with_2fa }
+    let(:same_address_as_id) { true }
+    let(:double_address_verification) { true }
+
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_capture_secondary_id_enabled).
+        and_return(true)
+      allow(IdentityConfig.store).to receive(:in_person_verify_info_controller_enabled).
+        and_return(true)
+      allow(user).to receive(:establishing_in_person_enrollment).
+        and_return(enrollment)
+    end
+
+    it 'displays expected headers & data on /verify_info',
+       allow_browser_log: true do
+        
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+      complete_prepare_step(user)
+      complete_location_step(user)
+      complete_state_id_step(
+        user, same_address_as_id: same_address_as_id,
+              double_address_verification: double_address_verification
+      )
+      click_idv_continue
+      complete_ssn_step(user)
+
+      # confirm url is /verify_info
+      expect(page).to have_current_path(idv_in_person_verify_info_path)
+
+      # verify page
+      expect(page).to have_content(t('headings.verify'))
+
+      # confirm headers are on template
+      expect(page).to have_content(t('headings.state_id').tr(' ', ' '))
```

Comment:
I'd like to point out that I added this back in. The heading in en.yml was updated recently. All places evaluating headings.state_id use tr (translate characters). My test was failing after I removed it.  I can look into why the need for the update in the last two weeks or alternatives to testing this.  (I can come back to this as I want to look at a few other items for this PR first) 

<img width="216" alt="Screenshot 2023-05-11 at 2 40 08 PM" src="https://github.com/18F/identity-idp/assets/125507397/f29bb9ff-28e7-4745-84f6-8142d716f287">

<img width="305" alt="Screenshot 2023-05-11 at 2 39 23 PM" src="https://github.com/18F/identity-idp/assets/125507397/7bed0ec8-fbe1-4531-8654-7f678fcd4817">




https://apidock.com/ruby/String/tr

Reply:
Ok, I tracked this down to [this comment](https://github.com/18F/identity-idp/pull/8277#discussion_r1177079517) and [this conversation](https://gsa-tts.slack.com/archives/C03N6KQBAKS/p1682533879701429). We added a non-breaking space to keep `State-issued ID` from being separated on a line break, and using `tr` seems to be the established way to get around the issues that causes in tests.